data/css/modules/card/_media.scss

```diff
@@ -1,35 +1,30 @@
 .CardMedia {
-    padding: 10px;
     background-color: white;
 
-    separator {
-        background-color: $black-20-color;
-        min-height: 1px;
-    }
-
     &__caption {
-        font-size: .81em;  /* 13pt * 1/12 * 0.75 DPI correction factor */
-        font-family: $display-font;
+        font-size: 20px;
```

Comment:
I think going `px` is fine here and lots of other places in a framework css dos the same.

The problem with `em` is since it cascades its a really bad fit for our modular setup. `2em` inside a module that set font size to be `1.5em` and your fonts are looking way different then in another modular hierarchy.

`rem` could be a possibility going forward, but its kinda useless unless we are doing some responsiveness with the the base font size which we are not. And that's out of scope for this issue anyway.